### PR TITLE
mention WinXP VS2017, LOADLIN latest

### PR DESCRIPTION
**Pull request by Torinde**
- automatic building of LOADLIN portable (last seen [0.83.25 2022-05-01](https://github.com/joncampbell123/dosbox-x/releases/tag/dosbox-x-v0.83.25)). If it's decided that this won't be build with every release, then INSTALL.md should link to the latest one.

_Originally posted by @Torinde in https://github.com/joncampbell123/dosbox-x/issues/3926#issuecomment-1386537129_

Mention in INSTALL.md the WinXP VS2017 builds and the latest version having LOADLIN.
Of course, would be also good if somebody can make the LOADLIN be automatically build like the rest of the versions.

### CODE REVIEW

**Comment by maron2000** [comment]
I suggest not using version numbers such as 2017 & 2022, since casual users will not find those number when downloading.
The current build is built using 2019, but maybe subject to change.

TL;DR
Visual Studio 2017 builds  -> Visual studio XP builds
Delete 2022, no version number required aince it is subject to change.

**Comment by Torinde** [comment]
Done.
